docs/en/remote-flow.rst

```diff
@@ -149,18 +150,22 @@ response, containing the request URI, are described in the Table below.
     - REQUIRED. Unique identifier of the Relying Party.
   * - **request_uri**
     - REQUIRED. The HTTPs URL where the Relying Party provides the signed Request Object to the Wallet Instance.
-  * - **client_id_scheme**
-    - OPTIONAL. The scheme used by the Relying Party for the client_id, detailing the format and structure and the trust evaluation method. It SHOULD be set with ``entity_id``.
   * - **state**
-    - OPTIONAL. A unique identifier for the current transaction generated by the Relying Party. The value SHOULD be opaque to the Wallet Instance.
+    - RECOMMENDED. A unique identifier for the current transaction generated by the Relying Party. The value SHOULD be opaque to the Wallet Instance.
   * - **request_uri_method**
```

Comment:
I think that the `request_uri_method` should be removed from the request object. The Wallet has already performed the GET/POST request to the  Request URI to obtain the Request Object, it does not need this info here.